File: curso-python/ex067.py
c = 0
while True:
    n = int(input("Quer ver a tabuada de qual valor?: "))
    if n < 0:
        print("PROGRAMA TABUADA ENCERRADO. Volte sempre!")
        break
    c = c + 1

    print("-"*20)
    for c in range(1,11):
        print(f"{n} x {c} = {n*c}")
    print("-"*20)

File: javascript-exercicios/ex067.js
const prompt = require("prompt-sync")()

let c = 0
while (true){
    let n = Number(prompt("quer ver a tabuada de qual valor: "))
    if(n < 0){
        console.log("programa tabuada encerrado. volte sempre!")
        break
    }
    c+=1

    console.log("-------------------------------------")
    for (let c=0; c < 11; c++){
        console.log(`${n} x ${c} = ${n*c}`)
    }
    console.log("-------------------------------------")

}